fhir-persistence-schema/src/main/resources/add_resource_type.sql

```diff
@@ -1,5 +1,5 @@
 -------------------------------------------------------------------------------
--- (C) Copyright IBM Corp. 2016
+-- (C) Copyright IBM Corp. 2016, 2020
```

Comment:
only copyright change?

Reply:
right, as I started with the existing branch